src/components/Interactive/MyFilter.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <div>
+    <div>
+      <h5>By Genre:</h5>
+      <my-tags/>
+    </div>
+    <div style="margin-top: 30px;">
+      <h5>By Rating:</h5>
+      <b-col sm="12">
+          <span class="multi-range">
+            <input type="range"
+                   @change="changeLeftVoteRating(leftRatingValue), getFilter({
```

Comment:
This is really ugly, make sure to avoid method-like code in templates.

Templates should be kept simple, as close to HTML as possible.

Reply:
Instead,
1. Create a method in this component.
2. Put all this code in that method, passing the template variables if needed to it.
3. Call a single on-change method, like `@change="changeRatingAndUpdateFilter"`, or similar.

Let me know if that makes sense.